applications/PGSAUT-abstract.md

```diff
@@ -0,0 +1,53 @@
+
+The presentation of this paper will itself follow participatory practices: it will be co-presented by an autistic participant, point attendees to open and transparent research protocols and data, and everyone will be invited to join as collaborators.
+
+We begin by exploring how existing scientific practices base themselves on a division between researchers and research subjects. 
+To do so, we explore Hacking’s account of human kinds and provide the example of autism. 
+We briefly review recent trends in the study of autism and argue that the gap traditionally conceived between “knowers” and those who are “known” raises ethical concerns.
+
+The ethical concerns we discuss are similar to those of feminist scholars at large and standpoint theorists in particular. 
+We will consider what Miranda Fricker ([2007](doi.org/10.1093/acprof:oso/9780198237907.001.0001)) calls “epistemic injustice”. 
+By Fricker’s account, prejudice about certain stereotypes render their testimony noncredible. 
+When these stereotypes are incorrect, an epistemic injustice is produced. 
+This is because social injustices are perpetuated through incorrectly assigned stereotypes. Despite the research subject being an expert of the own experience, research findings are sanctioned by scientific institutions and afforded authority by society. 
+This results in social policies and norms that further reproduce social injustices. Such divisions are further entrenched by societal inequality which can create barriers to scientific institutions and education.
+
+[Sue Fletcher-Watson et. al., (2019)](https://doi.org/10.1177/1362361318786721) further emphasise the cruciality of interrogating and redressing power disbalance in research. 
+Power asymmetry is associated with significant adverse effects for those disempowered. On an individual level, power asymmetry can lower self-esteem ([Wojciszke & Struzynska–Kujalowicz, 2007](https://doi.org/10.1521/soco.2007.25.4.472)), reduce the sense of agency ([Martin et al., 2003](#book-references)), and leads to reduced empathy in those wielding power, who can pathologize difference as dysfunction [Galinsky et al., 2006](https://doi.org/10.1111/j.1467-9280.2006.01824.x).
+
+Having problematised the division between “knowers” and “known”, we return to Hacking’s looping effects to argue that, given the self-reflecting nature of human kinds, scientists ought to integrate this into their practice. 
+In other words, given that researchers, in “knowing” about the social world, shape the very subject they study, they should fix the ethical principles guid their research from the start. 
+To both fix these ethical principles and study human kinds, we suggest that bringing research subjects into the research process is an adequate way to conduct science that is pertinent to human kinds.
```

Comment:
```suggestion
To both fix these ethical principles and study human kinds, we suggest that bringing research subjects into the research process is a crucial way to conduct science that is pertinent to human kinds.
```

Reply:
I would go further than 'adequate' here.

Reply:
I went for ethically compelling as there may be other solutions or exceptions.